packages/noya-state/src/selectors/layerSelectors.ts

```diff
@@ -95,3 +96,14 @@ export const addSiblingLayer = <
   const l = layer instanceof Array ? layer : [layer];
   parent.layers.splice(indexPath[indexPath.length - 1], 0, ...l);
 };
+
+export const getLeftMostLayerPoint = (page: Sketch.Page) => {
```

Comment:
Should this be called `getRightMostLayerBounds`? 

- Using `maxX` would give us the right edge. `minX` or just `frame.x` would give us left
- We're returning a Bounds type, instead of a Point